py-polars/polars/lazyframe/frame.py

```diff
@@ -5172,20 +5172,27 @@ def map_batches(
         --------
         >>> lf = pl.LazyFrame(
         ...     {
-        ...         "a": [1, 2],
-        ...         "b": [3, 4],
+        ...         "a": range(-100_000, 0),
```

Comment:
I don't really like that we have such an expensive test in our doctests. I get the idea of showing a more realistic example, but maybe a range of `1000` elements is enough? Or keep 100_000 and just skip this example?

Reply:
Good idea - adding the SKIP makes a lot of sense there; some of our examples are so trivial they're not really showing anything, so it's good to have something that feels at least a _little_ more "real" 😅

Reply:
Done 👍